sdk/websites/Azure.ResourceManager.AppService/src/Generated/WebSiteResource.cs

```diff
@@ -4564,6 +4564,64 @@ public virtual Response DeleteHostSecret(string keyType, string keyName, Cancell
             }
         }
 
+        /// <summary>
+        /// Description for Retrieves all Service Bus Hybrid Connections used by this Web App.
+        /// <list type="bullet">
+        /// <item>
+        /// <term>Request Path</term>
+        /// <description>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/hybridConnectionRelays</description>
+        /// </item>
+        /// <item>
+        /// <term>Operation Id</term>
+        /// <description>WebApps_GetAllTheHybridConnectionData</description>
+        /// </item>
+        /// <item>
+        /// <term>Default Api Version</term>
+        /// <description>2021-02-01</description>
+        /// </item>
+        /// <item>
+        /// <term>Resource</term>
+        /// <description><see cref="WebSiteResource"/></description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <returns> An async collection of <see cref="HybridConnectionData"/> that may take multiple service requests to iterate over. </returns>
+        public virtual AsyncPageable<HybridConnectionData> GetAllTheHybridConnectionDataAsync(CancellationToken cancellationToken = default)
```

Comment:
well this name reads weird...
Is `GetHybridConnections` taken?

Reply:
solved,changed the mthod name to 'GetHybridConnections '